### ISSUE TITLE
Downloading Alamofire via Cocoapods: Getting update Swift syntax. 

### ISSUE THREAD

**Issue by dannymout** [Alamofire/Alamofire#1683]
I am have Xcode 8 and using Swift 3. I am download Alamofire using the Podfile template provided in the [readme](https://github.com/Alamofire/Alamofire/blob/master/README.md) file. The problem I'm having, and I can't seem to understand is why I am getting popups to update to the latest Swift syntax. How do I fix this?

**Comment by orassr** [comment]
just press later ;)

**Comment by dannymout** [comment]
@orassr Doesn't work, it ask to set it to use a legacy swift version. I change the option to use a Legacy Swift Version, then I get 625 errors instantly pop up.

![http://i.imgur.com/xUtvE4j.png](http://i.imgur.com/xUtvE4j.png)

**Comment by zycoding** [comment]
I got the same issue after run "pod install" for installing other frameworks yesterday. Problem solved now. Below is what I did.
1. run the "pod update" command. This update my Alamofire from 4.0 to 4.0.1
2.open my project in xcode. It ask me to concert and I did. After concert to swift 3, get hundreds errors/warnings.
3. Don't bother with the errors. Just build and run the project and the error disappear.
